app/assets/javascripts/Components/Modals/submission_file_upload_modal.jsx

```diff
@@ -17,6 +17,21 @@ class SubmissionFileUploadModal extends React.Component {
 
   onSubmit = event => {
     event.preventDefault();
+    const newFilename = this.state.renameTo;
+    const originalFilename = this.state.newFiles[0].name; // Assuming only one file is uploaded
```

Comment:
This is a good thing to observe; you should *skip this check entirely* if there is more than one file being submitted. Note that in this case the "Renaming" functionality is also disabled.